src/PerformanceTests/Utils/SessionExtensions.cs

```diff
@@ -14,7 +14,7 @@ public static class SessionExtensions
         }
         catch (Exception ex)
         {
-            Log.Warn(ex, "CloseWithSuppress ({0}): {1}", ex.GetType(), ex.Message.Replace(Environment.NewLine,"; "));
+            Log.Warn(ex);
```

Comment:
Please add the message as previously build. This helps in filtering the log output.

Reply:
what does it bring to log the exception + a message which reformats the exception message as well?

Reply:
The location should be present in the log event, "CloseWithSuppress". As said, the formatting is so that the details are on a single line which is useful for filtering. The exception isn't logged on the same line thus when filtering it is not visible. As you can see, the exception message spans multiple lines. Azure is writing info over multiple lines.

If you feel strongly to not do the reformatting, then leave it out but do ass the "CloseWithSuppress" in the log event.